8_reference/73_techniques_best_practices.txt

```diff
@@ -25,6 +25,18 @@
 
 ==== Writing convention
 
+===== Technique Naming Guidelines
+
+The following rules should be followed when naming a new Technique:
+
+   * Try to keep names as short as possible, to improve readability
+   * Read the exinsting technique list, and particularly techniques related to what you are writing. The new names should be consistent with existing ones.
+   * The name should be a nominal group (not "Manage file content" or "Set Service state" but "File content" or "Service state)". It describes the target of the action, not the action itself.
```

Comment:
Always start with good examples to have bad examples later. Also, the phrasing here doesn't make it very clear what is good or bad.